lib/danger/commands/init.rb

```diff
@@ -121,6 +121,8 @@ def setup_access_token
         ui.pause 1
         ui.say "It's worth noting that you " + "should not".bold.white + " re-use this token for OSS repos."
         ui.say "Make a new one for those repos with just " + "public_repo".yellow + "."
+        ui.pause 1
+        ui.say "Additionally, don't forget to add your new GitHub account as a collaborator to your Closed Source project."
```

Comment:
It's being consistent: https://github.com/ffittschen/danger/blob/39ba22f8ed35e2a6fc527d5d2c371bd4ea7b1258/lib/danger/commands/init.rb#L118